core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CloudflowAkkaPlugin.scala

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cloudflow.sbt
+
+import sbt._
+import sbt.Keys._
+import sbtdocker._
+import sbtdocker.DockerKeys._
+import com.typesafe.sbt.packager.Keys._
+import spray.json._
+import cloudflow.sbt.CloudflowKeys._
+import cloudflow.blueprint.StreamletDescriptorFormat._
+import cloudflow.sbt.CloudflowKeys._
+
+object CloudflowAkkaPlugin extends CloudflowBasePlugin {
+
+  override def projectSettings = Seq(
+    libraryDependencies ++= Vector(
+      "com.lightbend.cloudflow" %% "cloudflow-akka-util" % BuildInfo.version,
+      "com.lightbend.cloudflow" %% "cloudflow-akka" % BuildInfo.version,
+      "com.lightbend.cloudflow" % "cloudflow-runner" % BuildInfo.version,
```

Comment:
Just curious why this one is not specified as a scala dep (%%)

Reply:
hmm .. Just wondering why this is so .. Checked up that we have this in the original version in pipelines as well. Any idea ?

Reply:
If `%%` works, make it so. If it doesn't, then that would be very strange and might need some investigation since AFAIK it is not a pure Java lib.

Reply:
would also prefer `%%`, please check if that works.

Reply:
We were stripping the runner jar of the version for Flink. Hence we used `%` .. Now that we have separate images, we can keep `%%` for Spark and Akka and keep `%` for Flink.